packages/preferences/src/browser/util/preference-types.ts

```diff
@@ -105,6 +105,18 @@ export namespace PreferencesCommands {
         id: 'preferences:copyJson.value',
         label: 'Copy Setting as JSON',
     };
+
+    export const OPEN_USER_PREFERENCES: Command = {
+        id: 'preferences:openUserPreferences',
```

Comment:
We should align with the same `id` as in vscode:

```suggestion
        id: 'workbench.action.openGlobalSettings',
```